lookout/style/format/features.py

```diff
@@ -82,7 +85,16 @@ def from_node(node: bblfsh.Node, file: str, path: str) -> Iterable["VirtualNode"
                               Position(*[f[1] for f in node.end_position.ListFields()]),
                               node=node, path=path)
             return
-        start_offset = outer_token.find(node.token)
+        token = node.token
```

Comment:
@m09 Can you please describe the problem which is solved here and give a few examples.

For sure this complicates the code so I need to figure out if there is a better fix.

Reply:
Sure. I'll start by describing the way we handle delimiters atm. When babelfish returns a `token` that is smaller that the piece of code spanned by the position of its node, we use the parts of the piece of code not covered by the node token as delimiters. That includes:

- `//` for `// It's a comment` (` It's a comment` is the token)
- `/*` and `*/` for `\* this is a block comment */` (` this is a block comment ` is the token)
- `"` and `"` for `"I'm a string"` (`I'm a string` is the token)

Now, sometimes, babelfish interprets what's in the delimiters. For example:

- `"a\"b"` will return `a"b` as a token, which means we can't find it directly in the piece of code covered by the node (the `\` is in the way)
- `1e2` will return `100.0` as a token (or `100` I'm not sure) which means once again we can't use the token information.

So it seems that we can't both have a general purpose delimiter finder based on the babelfish token and something that handles the case that interests us the most (strings, quotes are amongst our features) easily.

Reply:
Daaaaaaamn

Reply:
@juanjux @dennwc Is this smth you can easily fix or it comes directly from the native driver?

Reply:
just a clarification, I said the values were sometimes evaluated but they're actually always evaluated for SOME types, I'm not sure I was clear. So `StringLiteral` and `NumericLiteral` are always evaluated. Also, I found some instances of regexes that were parsed as strings (I'll try to reproduce if there's interest).